app/pulp/app/serializers/repository.py

```diff
@@ -36,11 +37,18 @@ class RepositorySerializer(ModelSerializer):
     )
     notes = NotesKeyValueRelatedField()
     scratchpad = ScratchpadKeyValueRelatedField()
+    importers = ImporterRelatedField(many=True)
```

Comment:
I think this qualifies:
https://github.com/pulp/pulp/pull/2975/files#diff-ddfd9f920c129f9da2b4263eed392ec1R135

The serializer is representing what's in the model, and the model explains what's going on. I think we aren't adding value by putting a comment about it here.